packages/integration-karma/test/wire/legacy-adapters/index.spec.js

```diff
@@ -164,4 +165,20 @@ describe('legacy wire adapters (register call)', () => {
             }, 0);
         });
     });
+
+    describe('with multiple same adapters and different configs', () => {
+        it('should invoke multiple wires when component is created', (done) => {
+            const elm = createElement('x-same-adapter-different-config', {
+                is: SameAdapterDifferentConfig,
+            });
+
+            setTimeout(() => {
```

Comment:
@yungcheng do we need the `setTimeout` in this test? the config for the wires is static, therefore, it will call update immediately.